stubs/D3DShot/@tests/requirements-stubtest.txt

```diff
@@ -0,0 +1 @@
+pillow
```

Comment:
If I recall correctly this needs to be an older version of Pillow, the newest release doesn't work.

Reply:
Yes, they have some complicated setup: https://pillow.readthedocs.io/en/latest/installation.html
<img width="700" alt="Снимок экрана 2022-10-19 в 21 10 15" src="https://user-images.githubusercontent.com/4660275/196771071-d3c45f40-decd-437d-b3f0-c00bc1865357.png">

I will try different ones.

Reply:
There's a weird dependency conflict here.
IIRC, D3DShot 0.13 didn't lock the pillow version, but 0.15 did.
And it locked it to 7.1.2 which is not 3.9 compatible.

There was a PR to fix that, but the repository got archived not long ago (most of the forks are just that change). Even more recently, most of SerpentAI's repos have been archived as well. The timing with the addition to Typeshed is unfortunate, (the stub was started before the repo got archived). Using D3DShot now is somewhat of a stopgap to let upcoming replacements mature. With an already planned removal (#8929), I think it may be acceptable to just keep `skip = true` in this stub, unless you can let pip ignore the requirement, or let it build from source with zlib.

Reply:
It is said that we need `Pillow>=7.2.0`: https://github.com/python/typeshed/pull/8652
But, we already have `Pillow>=7.2.0`, looks like we need an upper bound as well.
@Avasam, can you please take a look?

Reply:
What I didn't realize when I wrote that comment (https://github.com/python/typeshed/pull/8652#issuecomment-1237571064) is that requirering `Pillow>=7.2.0` will install D3DShot 0.1.3 .

https://github.com/SerpentAI/D3DShot/commit/a37abe4b6845246fe3437a9557f073b1b19f1963#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L18
Because pip's dependency resolver will see that 0.1.3 has `Pillow>=5.0.0` (which any newer version matches) while 0.1.4 is pinned at `Pillow~=7.1.2`

Reply:
No, it still does not work: https://github.com/sobolevn/typeshed/actions/runs/3284090937/jobs/5409620328

I think that existing deps can be re-installed here: https://github.com/python/typeshed/blob/master/tests/stubtest_third_party.py#L74

Reply:
Okay, I endorse @Avasam's suggestion of leaving D3DShot as `skip = true` for now -- this is all slightly tangential to the main point of this PR, and we also have `pywin32` to test that this PR works correctly on Windows. We can try to figure out how to get stubtest to install this package after this PR is merged :)